.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+** Please make sure you read the contribution guide and file the issues in the rigth place. **
+[Contribution guide.](https://google.github.io/adk-docs/contributing-guide/)
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Install '...'
+2. Run '....'
+3. Open '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```

Comment:
would be better to python version (python -V), adk version (pip show google-adk)